src/Grid/RemoteDataSource.js

```diff
@@ -89,39 +94,13 @@ class RemoteDataSource {
   }
   
   _doRequest(rowsPerPage, page, searchText) {
```

Comment:
This method shouldn't be "_doRequest" anymore, since the actual method doing the request is "getAllRecords"